migrations/2024-10-17-073555_add-profile-id-to-roles/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE roles ADD profile_id Varchar(64);
```

Comment:
nitpick:
- To keep drop.sql and up.sql consistent we can either use COLUMN keyword in both, or drop it for both, since it is optional.
- We can also use IF EXISTs and IF NOT EXISTs (Optional change)